silx/gui/hdf5/Hdf5Item.py

```diff
@@ -383,12 +383,16 @@ def nexusClassName(self):
                 text = text.strip('"')
                 # Check NX_class formatting
                 lower = text.lower()
+                formatedNX_class = None
```

Comment:
This results in `nexusClassName` that can be `None` while it was always `str` before.

From a quick look at the code it can lead to an issue here: https://github.com/silx-kit/silx/blob/617001f8fce5d39498869bc1153c88fe8bbc0dfc/silx/gui/hdf5/Hdf5Item.py#L426 where text is otherwise returned.

Maybe safer to initialize to empty string.

Reply:
That was one of my interrogation because for "def dataType(self, role)" can return `None`. And my interpretation of `nexusClassName` is more or less 'should be None' if unknow/undefined.
But I don"t know much this part of the source code.

Reply:
but if you think this is safer I can update it and test it on a file raising the error.

Reply:
I don't know much this code either and I would also be OK with `nexusClassName` to be None if no valid NX class.
Yet, when there is no a`NX_class` attribute, it is set to `''`: https://github.com/silx-kit/silx/blob/617001f8fce5d39498869bc1153c88fe8bbc0dfc/silx/gui/hdf5/Hdf5Item.py#L380
So I think it is better and safer to be consistent.

Reply:
yes you are right.